### PR TITLE
Added tests for remaining ListServiceWith* functions

### PR DESCRIPTION
**Pull request by eochoa260**

### CODE REVIEW

**Review by matthewbrahms — APPROVED** [review]

File: service_test.go [hunk 1]
```diff
@@ -289,6 +288,29 @@ func TestCreateService(t *testing.T) {
...
+	}`
+	response := `{"data": {"serviceUpdate": {
+     "service": {{ template "service_1" }},
+     "errors": []
+ }}}`
+
+	client := ABetterTestClient(t, "service/update", request, response)
+
+	// Act
+	_, err := client.UpdateService(ol.ServiceUpdateInput{
```
Comment (rocktavious) [inline service_test.go:305]: we should validate the response object has a correct value

**Review by rocktavious — CHANGES_REQUESTED** [review]
Just 1 adjustment on UpdateService test to ensure the deserializer works

File: service_test.go (hunk 1 above)
Comment (eochoa260) [reply service_test.go]: Oh yeah, missed that.  Just added one.

**Review by rocktavious — APPROVED** [review]
LGTM

**Pull request merged**